ansible/playbooks/sap-hana-cluster.yaml

```diff
@@ -5,6 +5,13 @@
   become_user: root
   vars:
     provider: azure # only provider for now
+    use_sbd: yes
```

Comment:
All the vars required for Azure fencing are added to the top of the playbook as empty.  The glue code will need to override these variables to use Azure fencing.  SBD remains the default method.